addon/index.js

```diff
@@ -19,6 +17,7 @@ export default function(str, formats) {
   return str.replace(/%@([0-9]+)?/g, function(s, argIndex) {
     argIndex = (argIndex) ? parseInt(argIndex, 10) - 1 : idx++;
     s = cachedFormats[argIndex];
-    return (s === null) ? '(null)' : (s === undefined) ? '' : inspect(s);
+    s = typeof s === 'string' ? s : inspect(s);
```

Comment:
These are the only changes required to get the test suite to pass. Only pass non-strings to `Ember.inspect` as it's behaviour changed between 1.x/2.x and 3.x.